data/tags.json

```diff
@@ -44,8 +44,14 @@
     "aliases": ["matrix"],
     "content": [
       "**Bridge** <:termux_smile:789347239591804950> <-> 🌉 <-> :termuwux:   ",
-      "To not force people to use one specific chat platform, we connected <#641256914684084237> and <#847704138711171084> channels to relay messages into Matrix: <https://matrix.to/#/#Termux:matrix.org>. It works through webhooks that pass messages between Discord and bridged rooms on Matrix, allowing us to hold conversations across multiple messaging platforms.",
```

Comment:
About the matrix space: I think this does need a list of channels. Not sure how useful linking the space is, and we'd need to update it for #coding. Thoughts welcome.